README.md

```diff
@@ -101,6 +102,7 @@ You can edit the following options values in the configuration file (~/myopenaps
 	"mode" = Optional - if you specify "expired" then mode is hard-coded to expired tx mode and it uses local LSR and single-point algorithms to calculate BG always.  If empty, or set to "not-expired", then native Dexcom algorithms will be used (i.e., BG is calculated by transmitter itself), if available, otherwise the local algorithms will be used. If you specify "native-calibrates-lsr" then it will use the Dexcom algorithms, unless not available, and it will also calibrate the local LSR algorithm based on the values of the Dexcom algorithm, every 6 hours. "native-calibrates-lsr" is most useful for G6 without manual calibrations. If you specify "read-only" then the behavior is the same as "not-expired" with the exception that all start/stop/calibrations must be done by the receiver. The default is "not-expired".
 	"pump_units" = Optional - pumpUnits default is "mg/dl"
 	"fake_meter_id" = Optional - meterid for fakemeter sending glucose records to pump, default is "000000"
+	"alternate_bluetooth_channel" = true or false. Optional - Default is false. If set to true then Logger uses the alternate channel to connect to the Dexcom transmitter. If set to true (the receiver cannot be used. However, when set to true, either the Xdip Plus android app or the Dexcom Iphone app can be used alongside Logger. Keep in mind that there is a higher chance for bluetooth conflict when connecting to the transmitter with both channels. You may be able to avoid some reconnects by keeping the rig and the phone app physically separated by several inches.
```

Comment:
This line has a unintended open parenthesis: `(`.

Reply:
Thanks - I'll correct it.